scripts/images/README.md

```diff
@@ -16,7 +16,7 @@
 - Install `python3` package
   - Install `jinja2` & `boto3` modules with `pip3`
   - Upgrade `PyYAML` module to v5.1 or newer; see https://stackoverflow.com/questions/55551191/module-yaml-has-no-attribute-fullloader
-- Install `qemu-img` package
+- Install `qemu-img cloud-utils-growpart` packages
```

Comment:
`cloud-utils-growpart` available in RHEL/CentOS 8.x ?

Reply:
yes!
```shell
[root@ip9-114-181-59 mkumatag]# yum info cloud-utils-growpart
Updating Subscription Management repositories.
Last metadata expiration check: 2:48:02 ago on Thu 15 Oct 2020 11:05:47 PM EDT.
Installed Packages
Name         : cloud-utils-growpart
Version      : 0.29
Release      : 3.el8
Architecture : noarch
Size         : 61 k
Source       : cloud-utils-growpart-0.29-3.el8.src.rpm
Repository   : @System
From repo    : rhel-8-for-ppc64le-appstream-rpms
Summary      : Script for growing a partition
URL          : https://launchpad.net/cloud-utils
License      : GPLv3
Description  : This package provides the growpart script for growing a partition. It is
             : primarily used in cloud images in conjunction with the dracut-modules-growroot
             : package to grow the root partition on first boot.

[root@ip9-114-181-59 mkumatag]#
```

Reply:
Thanks